### ISSUE TITLE
The Search text fields needs its label to be outside the search text field

### ISSUE THREAD

**Issue by subaha-cob** [CityOfBoston/Iterators#215]
Desired Result
• Labels for text fields are outside the text field

Actual Result
• The label for the search text field is inside the text field, which goes away once someone starts typing in the field.

Pages affected:

- [x] https://www.boston.gov/my-neighborhood?p2
- [x] https://www.boston.gov/pay-and-apply
- [x] https://www.boston.gov/cemeteries
- [x] https://www.boston.gov/community-centers
- [x] https://www.boston.gov/neighborhoods
- [x] https://www.boston.gov/parks-and-playgrounds
- [x] https://www.boston.gov/civic-engagement/boards-and-commissions
- [x] https://www.boston.gov/public-notices?title=The%20Lion%20-\
- [x] https://content.boston.gov/departments/food-access/boston-farmers-markets#map--443916

We need to add a label to each Input field to fix the issue.

`<label for="query" class="sf-i-l">Search</label>`

This is in response to @carolinestj flag about search button wonkiness.

**Comment by carolinestj** [comment]
@subaha-cob I think the main search on top of the nav also seems to have it

**Comment by carolinestj** [comment]
public notices when you search for something that doesn't have any results 
https://www.boston.gov/public-notices?title=The%20Lion%20-\

**Comment by subaha-cob** [comment]
@carolinestj please test this on UAT at:
 https://d8-uat.boston.gov/

**Comment by carolinestj** [comment]
@subaha-cob They all look good, thank you!
